components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/authz/OAuth2AuthzEndpoint.java

```diff
@@ -1710,6 +1715,21 @@ private String getSpTenantDomain(String clientId) throws InvalidRequestException
         }
     }
 
+    private String getLoginTenantDomain(OAuthMessage oAuthMessage, String clientId) throws InvalidRequestException {
+
+        if (!IdentityTenantUtil.isTenantedSessionsEnabled()) {
+            return getSpTenantDomain(clientId);
+        }
+
+        String loginTenantDomain =
+                oAuthMessage.getRequest().getParameter(FrameworkConstants.RequestParams.LOGIN_TENANT_DOMAIN);
+        if (StringUtils.isBlank(loginTenantDomain)) {
+            return getSpTenantDomain(clientId);
+        } else {
```

Comment:
else is redundant here. 
if (StringUtils.isBlank(loginTenantDomain)) {
            return getSpTenantDomain(clientId);
        }
retun loginTenantDomain

Reply:
Addressed with 7d8e344